### DISCUSSION TITLE
GHelper cmd popping up at startup (Windows 11 24H2)

### DISCUSSION THREAD

**Discussion by Doughkey** [seerge/g-helper#3444] [General]
Ever since updating to 24H2, I noticed that GHelper causes a cmd window to flicker when opening it (pressing the M4 key) during startup. it doesn't happen afterwards - only when opening GHelper after startup.

1. Power on laptop
2. After logging into windows, press m4 key
3. CMD window opens and closes in a blink of an eye

Device: Asus Tuf F15 FX507ZC4. RTX 3050 4GB VRAM, 512GB SSD, i12500H, Windows 11 24H2 latest stable build.

**Comment by seerge** [comment]
@Doughkey hello, G-Helper is not opening command prompt. Did you use official AC uninstaller tool to remove AC completely and did you stop remaining services from `Extra` settings?
